build.cake

```diff
@@ -4,7 +4,9 @@ var target = Argument("target", "Default");
 var configuration = Argument("configuration", "Release");
 
 Boolean.TryParse(EnvironmentVariable("TRAVIS"), out var travisEnabled);
```

Comment:
I think we should remove Travis flags since we no longer use it to avoid confusion